go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/ochinchina/supervisord
```

Comment:
It probably does not matter since no one is going to import this module. Still the right thing to do would be to rename the module to `github.com/cyralinc/supervisord`. Once you do that, because we don't want to fix the imports of the local packages, you'd also need a replace directive replacing `github.com/ochinchina/supervisord/XXX` to `./XXX` (can also be to `github.com/cyralinc/supervisord/XXX` but avoids some needless dependency maintenance IMO to just point to ./XXX).

Reply:
Also, we need to modify the go.mod files in the submodules. Please look for all go.mod files in the repo.

Reply:
Thanks for the heads up. The PR is now ready to be reviewed. Got it to work and tested it. We even have a basic CI now. :)